app/code/Magento/Config/Test/Unit/Model/Config/Backend/BaseurlTest.php

```diff
@@ -61,4 +69,47 @@ public function testSaveMergedJsCssMustBeCleaned()
         $model->setValue('http://example.com/')->setPath(Store::XML_PATH_UNSECURE_BASE_URL);
         $model->afterSave();
     }
+
+    /**
+     * @dataProvider beforeSaveDataProvider
```

Comment:
Add a detailed DocBlock for the method.

for e.g.
```
/**
 * Test beforeSave method to ensure URL is converted to lower case.
 *
 * @dataProvider beforeSaveDataProvider
 * @param string $value
 * @param string $expectedValue
 * @return void
 */
```

Reply:
@engcom-Hotel added detailed doc block.